consensus/polybft/fsm.go

```diff
@@ -19,7 +18,7 @@ import (
 )
 
 type blockBuilder interface {
-	Reset() error
+	Init() error
```

Comment:
Was it necessary to change this name?

Reply:
name has been reverted https://github.com/0xPolygon/polygon-edge/pull/1133/commits/4ccd5429fc4e2c8a2d3648f4b04e8aa43e20d007